Objects/Converters/ConverterRhinoGh/ConverterRhinoGhShared/ConverterRhinoGh.cs

```diff
@@ -679,7 +678,9 @@ public bool CanConvertToNative(Base @object)
       case Level _:
       case Text _:
       case Dimension _:
+      case Collection c when !c.collectionType.ToLower().Contains("model"):
```

Comment:
Can't we check for the `Model` class here instead?

Reply:
Unfortuantely no, `Model` isn't a class / subclass in use currently